Keen.NetStandard/AsyncLazy.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Keen.NetStandard
+{
+    // Simple version of AsyncLazy based on
+    // https://blogs.msdn.microsoft.com/pfxteam/2011/01/15/asynclazyt/
+    public class AsyncLazy<T> : Lazy<Task<T>>
```

Comment:
Or is that not possible since the compiler will say that technically you have a member property of a type that has less visibility than the property itself :S I bet that's it.